packages/cubejs-testing/birdbox-fixtures/smoke/schema/OrdersPA.js

```diff
@@ -1,25 +1,40 @@
+import { CUBEJS_QUERY_TYPE } from '../env';
+import { DATE_CAST } from './CAST';
+
+// TODO(cristipp) Use CUBEJS_DB_TYPE instead, once DATE_CAST['bigquery'] gets fixed.
+const { DATE_PREFIX, DATE_SUFFIX } = DATE_CAST[CUBEJS_QUERY_TYPE];
+
 cube(`OrdersPA`, {
   sql: `
-    select 1 as id, 100 as amount, 'new' status
+    select 1 as id, 100 as amount, 'new' as status, ${DATE_PREFIX}'2020-01-10'${DATE_SUFFIX} as date
```

Comment:
I wonder why you made this most straightforward schema more complicated instead of using a more complex ECommerce schema. It contains the date field and can be used for testing your scenarios. So, maybe it will be a better option here?